models/comments-models.js

```diff
@@ -12,6 +12,16 @@ exports.selectCommentsByArticleId = (articleId) => {
     });
 };
 
+exports.deleteComment = (comment_id) => {
```

Comment:
Your model and controller are named the wrong way round. The convention is the controller should include the method (delete in this case) and the model should use a verb that means the same thing eg "remove".